dashboard/app/controllers/courses_controller.rb

```diff
@@ -30,23 +30,6 @@ def new
   end
 
   def show
-    if current_user&.user_type == "teacher" && current_user.sections_instructed.any? {|s| s.course_id == @unit_group.id} && (Experiment.enabled?(user: current_user, experiment_name: 'teacher-local-nav-v2') || DCDO.get('teacher-local-nav-v2', false))
-      section_id = params[:section_id] || current_user.sections_instructed.select {|s| s.course_id == @unit_group.id}.last.id
-      if section_id
-        if @unit_group.single_unit_course?
-          redirect_to "/teacher_dashboard/sections/#{section_id}/unit/#{@unit_group.default_units.first.name}"
```

Comment:
Instead of this custom logic, we can rely on the redirect to the `script` page which has it's own redirect to the teacher dashboard. This is technically 2 redirects instead of just one, however it is safer to use the script endpoint logic.